test/tests/ver-1kb/comparison_ver-1kb.py

```diff
@@ -0,0 +1,113 @@
+expt_data = pd.read_csv(csv_folder)
+TMAP8_time = expt_data['time']
+TMAP8_pressure_enclosure_1 = expt_data['pressure_enclosure_1']
+TMAP8_pressure_enclosure_2 = expt_data['pressure_enclosure_2']
+concentration_enclosure_1_at_interface = expt_data['concentration_enclosure_1_at_interface']
+pressure_enclosure_2_at_interface = expt_data['pressure_enclosure_2_at_interface']
+mass_conservation_sum_encl1_encl2 = expt_data['mass_conservation_sum_encl1_encl2'].values
+concentration_ratio = expt_data['concentration_ratio']
+
+# Repeat the same for K=10/RT
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
+    csv_folder_k10 = "../../../../test/tests/ver-1kb/gold/ver-1kb_out_k10.csv"
+else:                                  # if in test folder
+    csv_folder_k10 = "./gold/ver-1kb_out_k10.csv"
+expt_data_k10 = pd.read_csv(csv_folder_k10)
+TMAP8_time_k10 = expt_data_k10['time']
+TMAP8_pressure_enclosure_1_k10 = expt_data_k10['pressure_enclosure_1']
+TMAP8_pressure_enclosure_2_k10 = expt_data_k10['pressure_enclosure_2']
+concentration_enclosure_1_at_interface_k10 = expt_data_k10['concentration_enclosure_1_at_interface']
+pressure_enclosure_2_at_interface_k10 = expt_data_k10['pressure_enclosure_2_at_interface']
+mass_conservation_sum_encl1_encl2_k10 = expt_data_k10['mass_conservation_sum_encl1_encl2'].values
+concentration_ratio_k10 = expt_data_k10['concentration_ratio']
+
+# Subplot 1: Time vs Pressure
+fig1 = plt.figure(figsize=[6, 5.5])
+ax1 = fig1.add_subplot(111)
+ax1.plot(TMAP8_time / 3600, TMAP8_pressure_enclosure_1, label=r"T2 Encl 1", c='tab:red', linestyle='-')
+ax1.plot(TMAP8_time / 3600, TMAP8_pressure_enclosure_2, label=r"T2 Encl 2", c='tab:blue', linestyle='-')
+ax1.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+ax1.set_xlabel('Time (hr)')
+ax1.set_ylabel('Pressure (Pa)')
+ax1.set_xlim(0, 3)
+ax1.set_ylim(bottom=0)
+ax1.legend(loc="best")
+ax1.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig1.savefig('ver-1kb_comparison_time.png', bbox_inches='tight')
+
+# Subplot 2: Solubility vs time
+fig4 = plt.figure(figsize=[6, 5.5])
+ax4 = fig4.add_subplot(111)
+ax4.plot(TMAP8_time[1:] / 3600, concentration_ratio[1:], label=r"TMAP8", color='tab:blue', linestyle='-')
+ax4.plot(TMAP8_time[1:] / 3600, [1] * len(TMAP8_time[1:]), label=r"Analytical", color='tab:red', linestyle='--')
+ax4.set_yticks(np.arange(0, 3, 1))
+ax4.set_xlim(0,TMAP8_time.max() / 3600)
+ax4.set_xlabel('Time (hr)')
+ax4.set_ylabel(r"Concentrations ratio $C_{\text{encl1}} / C_{\text{encl2}}$")
+ax4.legend(loc="best")
+ax4.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig4.savefig('ver-1kb_concentration_ratio.png', bbox_inches='tight')
+
+# Subplot 3: Mass Conservation Sum Encl 1 and 2 vs Time
+fig3 = plt.figure(figsize=[6, 5.5])
+ax3 = fig3.add_subplot(111)
+ax3.plot(TMAP8_time / 3600, mass_conservation_sum_encl1_encl2, 'o', color='tab:blue', markersize=0.7)
```

Comment:
```suggestion
ax3.plot(TMAP8_time / 3600, mass_conservation_sum_encl1_encl2, c='tab:blue')
```
Why user markers here?